src/components/views/rooms/EventTile.js

```diff
@@ -152,6 +160,32 @@ module.exports = React.createClass({
         return false;
     },
 
+    componentWillUnmount: function() {
+        dispatcher.unregister(this.dispatcherRef);
+    },
+
+    onAction: function(payload) {
+        switch (payload.action) {
+            case 'device_verified':
```

Comment:
Hum; I hadn't thought about that. The problem is that a naive approach means that the HS could fiddle with the list of devices you trust. I suppose we need this, but there would have to be some sort of signing. Either way, it's not going to happen for a while.

I guess it makes sense to raise this as a js-sdk event anyway.